app/controllers/games_controller.rb

```diff
@@ -1,13 +1,25 @@
 # Main controller for chess session for CRUD
 # logic for our app
 class GamesController < ApplicationController
+  before_action :authenticate_user!, only: [:create, :new]
+  def index
+  end
+
   def create
-    @game = Game.new
+    @game = current_user.game.create(game_params)
```

Comment:
We didn't set up any associations with users and I don't see one in this PR so I would not expect this line to work.  There should also be a test for this because it would come back failing.